lib/listener.js

```diff
@@ -48,7 +48,7 @@ function createListener(url, options) {
    *
    * @param {String} exchange Exchange name
    */
-  function* listen(exchange) {
+  function* listen(exchange, opts) {
```

Comment:
please update the jsdoc to reflect this change (and maybe the readme if relevant)